configs/components/openssl.rb

```diff
@@ -13,7 +13,7 @@
     # needed for the makedepend command
     pkg.build_requires 'imake' if platform.name =~ /^el/
     pkg.build_requires 'xorg-x11-util-devel' if platform.name =~ /^sles/
-    pkg.build_requires 'xutils-dev' if platform.is_huaweios?
+    pkg.build_requires 'xutils-dev' if platform.is_deb?
```

Comment:
@stahnma looks like these arm platforms haven't been built since we made the default switchover to ruby 2.3.3 - the build blows up immediately due to not being able to find the rbconfig file for the platform. That's something else that needs to be fixed. 

Both of these arm platforms are debian-8 only, so I have verified that our debian 8 amd64 pooler hosts do not have the makedepend command by default and installing xutils-dev will get you that command. IMO the worst case scenario would be that this would just pull in an extra package that might not be used.